extensions/mssql/src/mssql.d.ts

```diff
@@ -943,6 +944,36 @@ declare module 'mssql' {
 			 */
 			schema: string | undefined;
 		}
+
+		export interface Database extends SqlObject {
+			owner?: string;
+			collationName?: string;
+			recoveryModel?: string;
+			compatibilityLevel?: string;
+			containmentType?: string;
+		}
+
+		export interface DatabaseViewInfo extends ObjectViewInfo<Database> {
+			loginNames: string[];
+			collationNames: string[];
+			compatibilityLevels: string[];
+			containmentTypes: string[];
+			recoveryModels: string[];
+		}
+
+		export interface ServerPropertiesInfo extends SqlObject {
```

Comment:
IT is better to get latest main branch to yours, these props have already moved to new file.